README.md

```diff
@@ -61,6 +61,12 @@ MY_WEIRD_ENV_VAR=123_ABC #required,format=\d{3}_\w{3}
 
 In the above example, `\d{3}_\w{3}` is converted to a regexp and the value is checked against it.
 
+## If you use Docker, read this
+
+Docker automatically reads `.env` files present in the project's root. What this means is that, if you use `dotenv_validator` in an app you run in a docker container, you might get exceptions or warnings about your variables being in the wrong format even though they're right. The reason is that Docker parses the `.env` file and sets the `ENV` variable, but it isn't as careful when dealing with comments present in the file and these might end up getting inserted into your environment variables, throwing off `dotenv_validator`.
+
+Unfortunately, there is no way around this as of yet. So should you see any errors related to a wrong parsing of your `.env` file, the fix is to simply remove the validating comments from the file. **DO NOT** remove them from `.env.sample`, but make sure your `.env` (or whatever it is you named your env file) has none.
```

Comment:
I would remove the `(or whatever it is you named your env file)`, the issue is present if the file is `.env` because docker-compose loads that automatically. Actually, a possible workaround maybe would be to use a different filename instead of `.env` so docker-compose don't read that by default

Naming the file `.env.local` or `.env.development` instead `.env` may work since docker-compose will probably ignore it but the Dotenv gem will understand it out of the box https://github.com/bkeepers/dotenv#what-other-env-files-can-i-use

Reply:
```suggestion
Unfortunately, there is no way around this as of yet. So should you see any errors related to a wrong parsing of your `.env` file, the fix is to simply remove the validating comments from the file. **DO NOT** remove them from `.env.sample`, but make sure your `.env` has none.
```